deploy/foreman-tasks.sysconfig

```diff
@@ -6,3 +6,15 @@ FOREMAN_LOGGING=warn
 FOREMAN_LOGGING_SQL=warn
 FOREMAN_TASK_PARAMS="-p foreman"
 FOREMAN_LOG_DIR=/var/log/foreman
+
+#Set the number of executors you want to run
+#EXECUTORS_COUNT=1
+
+#Set memory limit for executor process, before it's restarted automatically
+#EXECUTOR_MEMORY_LIMIT=1gb
+
+#Set delay before first memory polling to let executor initialize (in sec)
+#EXECUTOR_INIT_DELAY=180
+
+#Set memory polling interval, process memory will be checked every N seconds.
+#EXECUTOR_POLLING_INTERVAL=60
```

Comment:
I understand that you want to bring the memory back as soon as possible, but the process must survive hours, therefore increasing periods to minutes (or tens of minutes) should not be a problem. The golden rule of Ruby process restarting is 24 hours, if you need to restart your processes more frequently, then you should spend your time on fixing the issues rather than restarting them more often. All I want to say is let's prevent restarting-in-a-loop.